examples/offline_inference_kv_cache_verification.py

```diff
@@ -0,0 +1,177 @@
+# SPDX-License-Identifier: Apache-2.0
+"""
+This script performs an automated correctness verification for the TPUOffloadConnector.
+
+The verification works by performing a two-stage experiment for multiple prompts:
+1.  Baseline Run: For each prompt, it first runs a text generation using a
+    standard vLLM engine configuration without any KV cache connector. The
+    output from this run is considered the "source of truth".
+
+2.  Test Run: It then runs the exact same text generation, but this time
+    with the TPUOffloadConnector enabled via the `--kv-transfer-config` argument.
+    It runs the generation twice to verify prefix caching.
+
+3.  Comparison: The script compares the output from each test run against the
+    output from the baseline run for that prompt.
+
+The script succeeds (exits with code 0) only if the generated text is
+bit-for-bit identical in all runs for all prompts. A fixed seed is used to
+ensure that the generation process is deterministic and the comparison is
+valid. If any output differs, it raises an error, causing the script to fail
+(exit with a non-zero code).
+"""
+
+import copy
+import os
+import time
+from typing import List, Tuple
+
+import vllm.envs as envs
+from vllm import LLM, EngineArgs, SamplingParams
+from vllm.utils import FlexibleArgumentParser
```

Comment:
This needs to be updated to `from vllm.utils.argparse_utils import FlexibleArgumentParser` The docker image now is pulling from the vllm main branch, and lib got moved to another place